docs/contributing/assets/allowed_actions.json

```
@@ -140,6 +140,17 @@
         "repository": "https://github.com/suzuki-shunsuke/tfcmt"
       },
       "comment": ""
+    },
+    {
+      "name": "addnab/docker-run-action",
+      "versions": [
+        "v3"
```

```suggestion
        "4f65fabd2431ebc8d299f8e5a018d79a769ae185"
```
The approval is bound to a specific version of an action and to bind the action to this version it is necessary to use the commit sha of the version instead of the version tag. Especially if the version tag contains only the major version.